examples/GetNPUsers.py

```diff
@@ -318,9 +318,11 @@ def run(self):
     def request_users_file_TGTs(self):
 
         with open(self.__usersFile) as fi:
-            usernames = [line.strip() for line in fi]
+          def _usernames(fi):
+            for line in fi:
+              yield line
 
-        self.request_multiple_TGTs(usernames)
+          self.request_multiple_TGTs(_usernames(fi))
```

Comment:
The coding style of the module appears to use 4 spaces per level of indent, please do the same.